### ISSUE TITLE
Italian translation

### ISSUE THREAD

**Issue by lorenzovngl** [Piano-Walrus/termCalc-app#1]
Hi! I tried this app on my smartphone and I found it really interesting, so I was wondering if a translation in Italian could be a good thing for it. If yes, I can work on a localization for Italy.

**Comment by Piano-Walrus** [comment]
Hey! Thanks so much for your interest in the project! If you'd like to volunteer, you're free to make a pull request with Italian Translations (:

Most of the app's text is in app/src/main/res/values-it/strings.xml (with the exception of quite a few hard-coded strings scattered around the app since I'm still in the process of updating old lazy code I wrote when I first started learning Java, haha). If you'd like to add Italian Translations, just make a pull request editing that file (as I've just created it for this exact purpose, lol). My only request is that anything marked "translatable=false" stay untranslated. Much of this "untranslatable" text is either rarely seen, not meant to be seen by end-users, literally untranslatable, or frequently changing and thus not viable to translate.
